tests/ctest/test_cmerge.c

```diff
@@ -0,0 +1,581 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <kdbmerge.h>
+#include <tests.h>
+
+#define default_result_size 30 // Enough space for possible strange results
+#define OUR_ROOT "user/our"
+#define THEIR_ROOT "user/their"
+#define BASE_ROOT "user/base"
+#define RESULT_ROOT "user/result"
+#define OUR_KEY1 "user/our/key1"
+#define THEIR_KEY1 "user/their/key1"
+#define BASE_KEY1 "user/base/key1"
+#define RESULT_KEY1 "user/result/key1"
+#define ORIGINAL_VALUE "1"
+#define CHANGED_VALUE "2"
+#define MORE_CHANGED_VALUE "3"
+#define COMMENT "comment"
+#define SOME_COMMENT "some_comment"
+#define OTHER_COMMENT "other_comment"
+#define OTHER_COMMENT_LENGTH 13
+// This is arbitrarily chosen.
+// In test cases were no conflict should occur the strategy is irrelevant.
+#define MERGE_STRATEGY_IRRELEVANT 1
+
+void printKs (KeySet * ks)
```

Comment:
I think a similar function already exist in the testing framework.

Reply:
Couldn't find it in the cframework folder, only compare_keyset

Reply:
output_key output_keyset

Reply:
Ahh, now I see. I actually copied this function from libs/merge/kdbmerge.c where I implemented it first for debugging. I'll delete it.